docs/static/doc-for-plugin.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[plugin-doc]]
+=== Documenting your plugin
```

Comment:
We really need to change the generator tool to spit out a file. In the mean time, we should point users to the example plugins for an example of how to create the doc file. For example:  https://github.com/logstash-plugins/logstash-filter-example/blob/master/docs/index.asciidoc

Reply:
This one never got merged, so we should make sure it's correct and merge it: https://github.com/logstash-plugins/logstash-codec-example/pull/9

Reply:
Here's the PR to generate a placeholder asciidoc file:  #8187

Reply:
PR for codec-example is https://github.com/logstash-plugins/logstash-codec-example/pull/10.  At the moment it's failing CI because it doesn't contain buildable code.